test-data/unit/check-classes.test

```diff
@@ -4115,7 +4115,7 @@ def f() -> type: return M
 class C1(six.with_metaclass(M), object): pass  # E: Invalid base class
 class C2(C1, six.with_metaclass(M)): pass  # E: Invalid base class
 class C3(six.with_metaclass(A)): pass  # E: Metaclasses not inheriting from 'type' are not supported
-@six.add_metaclass(A)  # E: Metaclasses not inheriting from 'type' are not supported
+@six.add_metaclass(A)  # E: Metaclasses not inheriting from 'type' are not supported  # E: Argument 1 to "add_metaclass" has incompatible type "Type[A]"; expected "Type[type]"
```

Comment:
One more thing: Add `\` somewhere here and format it nicely with a newline, like this:
```
failing_func(wrong) # E: Bad function \
                    # E: Bad argument
```